src/inspector_profiler.cc

```diff
@@ -320,6 +307,7 @@ void StartProfilers(Environment* env) {
   }
   if (env->options()->cpu_prof) {
     const std::string& dir = env->options()->cpu_prof_dir;
+    env->set_cpu_prof_interval(env->options()->cpu_prof_interval);
```

Comment:
We could just use `env->options()->cpu_prof_interval` everywhere instead of an extra setter/getter pair, right?

Reply:
I was thinking about making this profiling capability available on-demand during runtime in the future, as requested [here](https://github.com/nodejs/node/issues/27421#issuecomment-486948017), in that case I think it makes more sense to put the options on the Environment itself (or in a separate struct when we have too many of them), because these may not come from CLI at that point.